src/components/App.tsx

```diff
@@ -57,7 +57,12 @@ const App: React.FC = (): JSX.Element => {
       <NodeView
         node={currentState.tree}
         dispatch={dispatch}
-        store={currentState}
+        justifications={currentState.justifications}
+        feedbackMap={
```

Comment:
We could remove the ternary and just have `feedbackMap={currentState.feedback.feedback}`, if feedback only exists on the `currentState.feedback object` if `currentState.feedback.success` is true.

Reply:
You're right in terms of runtime behaviour, but it doesn't type check because you're not supposed to access properties that don't exist on the object's type.
The ternary convinces the type-checker that it's okay to access 'feedback' if `success` is true.
Buuuut... I can add `feedback: undefined` to the failure case of the type and it will work.

I'll probably try to simplify the type more, since this is the only place I ended up checking the `success` discriminant.